client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,986 @@
+				),
+				tooltip: translate( 'feature explanation' ),
+				planOne: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
+				planTwo: {
+					included: true,
+					copy: translate( '58,000+ Available' ),
+					mobileCopy: translate( '58,000+ plugins available' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( '58,000+ Available' ),
+					mobileCopy: translate( '58,000+ plugins available' ),
+				},
+			},
+			{
+				featureName: translate( 'Install custome themes' ),
+				tooltip: translate(
+					'Themes change the design of your site. We provide dozens of professional free themes for a wide range of uses. On the Business and eCommerce plan, you can also install any 3rd party WordPress theme, free or paid, from the 8,000 custom themes made by 3rd party designers.'
+				),
+				planOne: {
+					included: false,
+					copy: translate( 'Not included' ),
+					mobileCopy: null,
+				},
+				planTwo: {
+					included: true,
+					copy: translate( '8,000 to choose from' ),
+					mobileCopy: translate( '8,000 themes to choose from' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( '8,000 to choose from' ),
+					mobileCopy: translate( '8,000 themes to choose from' ),
+				},
+			},
+			{
+				featureName: translate( 'eCommerce' ),
+				tooltip: translate(
+					'Build, manage, and scale your online store with unlimited products or services, multiple currencies, integrations with top shipping carriers, and eCommerce marketing tools.'
+				),
+				planOne: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
+				planTwo: {
+					included: false,
+					copy: translate( 'Not included' ),
+					mobileCopy: null,
+				},
+				planThree: {
+					included: true,
+					copy: translate( 'Complete eCommerce platform' ),
+					mobileCopy: translate( 'Complete eCommerce platform' ),
+				},
+			},
+			{
+				featureName: translate( 'Custom domain name' ),
+				tooltip: translate(
+					'Remove “.wordpress.com” from your site address with a custom domain of your choosing. Free for one year (will renew at its regular price).'
+				),
+				planOne: {
+					included: true,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+				planTwo: {
+					included: true,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+			},
+			{
+				featureName: translate( 'SEO' ),
+				tooltip: translate(
+					'SEO (Search Engine Optimization) helps your site rank higher, and more accurately in search engines, so more people can find your site.'
+				),
+				planOne: {
+					included: true,
+					copy: translate( 'Simple' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **38** times:
`translate( 'Simple', { context: 'Theme Showcase term name'} )` *ES Score: 13*